R/plot-individual-time-profile.R

```diff
@@ -58,7 +58,8 @@ plotIndividualTimeProfile <- function(dataCombined,
 #' @noRd
 .plotTimeProfile <- function(dataCombined,
                              defaultPlotConfiguration = NULL,
-                             quantiles = NULL) {
+                             aggregation = NULL,
+                             ...) {
   # validation -----------------------------
```

Comment:
Here we should check that `aggregation `is one of the allowed enum values

Reply:
also answering for: https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/1263#issuecomment-1611096224

This is done at a lower level, directly in the .extractAggregatedSimulatedData() function here: 
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/68b73fb6d03a7a7c34a7a9f56054d2d532a7ddf4/R/utilities-plotting.R#L149

Here's what it looks like:

```r
plotPopulationTimeProfile(myDataComb,
                            aggregation = "anotherAggregation")
# Error in `.extractAggregatedSimulatedData()` at OSPSuite-R/R/plot-individual-time-profile.R:111:4:
#   ! `aggregation` must be one of "quantiles", "arithmetic", or "geometric", not "anotherAggregation".
# Run `rlang::last_trace()` to see where the error occurred.
```

Reply:
I see. Still, the enum is preferable from my point of view.

Reply:
done in [be9d183](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/1263/commits/be9d1836253d77ba30804e17a77f813be895ebaa). 

I created a `SimulatedDataAggregationMethods` `ospsuite.utils::enum` and used `ospsuite.utils::validateEnumValue` for validation